src/Symfony/Component/HttpKernel/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ CHANGELOG
  * Add `#[WithHttpStatus]` for defining status codes for exceptions
  * Use an instance of `Psr\Clock\ClockInterface` to generate the current date time in `DateTimeValueResolver`
  * Add `#[WithLogLevel]` for defining log levels for exceptions
+ * Extract the `ErrorHandler` configuration logic into a separate service
```

Comment:
this is all internal classes, no need for this line

Reply:
Removed